docs/learn/learn-account-abstraction.md

```diff
@@ -0,0 +1,114 @@
+---
+id: learn-account-abstraction
+title: Polkadot's Account Abstraction
+sidebar_label: Account Abstraction
+description: Polkadot's Native Account Abstraction.
+keywords: [account, polkadot account, account abstraction, proxy, multisig, batch]
+slug: ../learn-account-abstraction
+---
+
+## Your Keys, Your Responsibility
+
+Account abstraction addresses the challenges of managing cryptographic keys representing accounts on
+blockchains. Accounts on blockchains represent entities, from an individual's identity to an
+institution. In Web3, you digitally sign any transaction or, more generally, any message using your
+private key. Data is recorded on a public ledger (usually blockchain-based) whose multiple copies of
+it are stored in computers participating in a P2P network.
+
+While the account’s private keys grant users control and ownership, losing them results in losing
+access to digital assets and fragmentation of your digital identity since you will need to create a
+new account with a new set of keys. This poses a hurdle for both users and developers regarding
+security and adoption.
+
+## Definition of Account Abstraction
+
+The term _abstraction_ refers to separating the user experience from the private key, enabling a
+piece of code to dictate account behavior. This allows for increased flexibility of accounts that
+originally were not engineered to be flexible and decreased chances of key mismanagement.
+
+Users are still responsible for their keys, but through account abstraction, they can take
+precautions to ensure they do not end up losing their accounts.
+
+Account abstraction introduces a layer of on-chain logic that controls an account, typically in the
+form of a smart contract. Without a smart contract, abstracting accounts would require changes in
+the core architecture of the protocol.
+
+The highly generic codebase of Polkadot's architecture makes accounts natively flexible and abstract
+without the direct need for smart contracts.
+
+## Origin Abstraction in Polkadot
+
+Adopting a generic design is crucial in scaling Web3 technologies. Abstraction and generalization of
+protocols are essential to improving user experience and security in blockchain adoption.
+
```

Comment:
Not sure if this counts as 'opinionated' for Wiki contenxt, although I think it's nice to include